### PR TITLE
[BEAM-1066] Add a test of ReleaseInfo

### PR DESCRIPTION
**Pull request by dhalperi**
R: @lukecwik OR @kennknowles

### CODE REVIEW

File: sdks/java/core/src/test/java/org/apache/beam/sdk/util/ReleaseInfoTest.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    ReleaseInfo info = ReleaseInfo.getReleaseInfo();
+
+    // Validate name
+    assertThat(info.getName(), containsString("Beam"));
+
+    // Validate semantic version
+    String version = info.getVersion();
+    String pattern = "\\d+\\.\\d+\\.\\d+.*";
+    assertTrue(
+        String.format("%s does not match pattern %s", version, pattern),
```
Comment (kennknowles) [inline sdks/java/core/src/test/java/org/apache/beam/sdk/util/ReleaseInfoTest.java]: Can we scrape the version in a different way here, to make sure they stay in sync?

File: sdks/java/core/src/test/java/org/apache/beam/sdk/util/ReleaseInfoTest.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
+package org.apache.beam.sdk.util;
```
Comment (lukecwik) [inline sdks/java/core/src/test/java/org/apache/beam/sdk/util/ReleaseInfoTest.java]: Missing license

**Comment by dhalperi** [comment]
This is failing in travis. Hmm. https://travis-ci.org/dhalperi/incubator-beam/jobs/180461943#L569

**Comment by kennknowles** [comment]
Interesting. Succeeding only in Eclipse-JDT.

**Comment by dhalperi** [comment]
(Eclipse-JDT skips tests ;)

**Comment by kennknowles** [comment]
Ha. I see.

**Comment by dhalperi** [comment]
This now passes in Jenkins as well as Travis (from my copy: https://travis-ci.org/dhalperi/incubator-beam/builds/180493809)

**Pull request closed without merging**